### DISCUSSION TITLE
[DOCS] x-on .dot modifier missing in documentation

### DISCUSSION THREAD

**Discussion by benjaminfayaz** [alpinejs/alpine#2251] [3. Feature ideas]
The `.dot` modifier when using `x-on` (or `@`) is useful when combined with e.g. bootstrap events, which might contain dots.
Since `@show.bs.collapse="menuIsOpen = true"` does not work, one can use `@show-bs-collapse.dot="menuIsOpen = true"`.

This should be included in the documentation. Currently it can only be found by looking at the source code.

**Comment by SimoTod** [comment]
https://github.com/alpinejs/alpine/blob/main/packages/docs/src/en/directives/on.md the relevant section is here if you want to send a PR

**Reply by benjaminfayaz** [reply]
https://github.com/alpinejs/alpine/pull/2253 🚀
